heron/common/tests/java/com/twitter/heron/common/config/Constants.java

```diff
@@ -23,9 +23,6 @@
 
   public static final String LAUNCHER_CLASS_KEY = "heron.launcher.class";
 
-  public static final String TEST_DATA_PATH =
```

Comment:
While you're here would you please remove `public` from all these constants (and the class) and remove GROUP_KEY and VERSION_KEY, which are unused. #cruftslayer